### PR TITLE
Prevent indefinite pending unregister

### PR DESCRIPTION
**Pull request by itsisak**
If `EventHasClosed` exception happens, the registration would stay as pending forever. 

Currently the unregister-button is not displayed correctly when the event has seperate registration and unregistration deadlines so it is possible for this to happen. I am fixing that in frontend so it should not be possible anymore, but the registration should never be able to stay as pending imo anyways.

### CODE REVIEW

**Review by jonasdeluna — APPROVED** [review]
Awesome!

File: lego/apps/events/tasks.py [hunk 1]
```diff
@@ -145,6 +145,13 @@ def async_unregister(self, registration_id, logger_context=None):
             exception=e,
             registration_id=registration.id,
         )
+        registration.status = constants.FAILURE_UNREGISTER
+        registration.save()
+        notify_user_registration(
+            constants.SOCKET_UNREGISTRATION_FAILURE,
+            registration,
+            error_message="Avregistrering feilet",
```
Comment (ivarnakken) [inline lego/apps/events/tasks.py:153]: Wouldn't hurt to add why in the error message as well hehe

**Review by ivarnakken — APPROVED** [review]
Very nice!!

File: lego/apps/events/tasks.py (hunk 1 above)
Comment (itsisak) [reply lego/apps/events/tasks.py:153]: Fair, just copied from the other exception

**Review by eikhr — APPROVED** [review]

**Review by ch0rizo — COMMENTED** [review]
LGMT!

**Pull request merged**